### DISCUSSION TITLE
What is the status of bfe?

### DISCUSSION THREAD

**Discussion by liangyuanpeng** [bfenetworks/bfe#1137] [General]
Hi guys, first of all I think this project is amazing and that's why I would like to know what the status of this project is and if you intend to keep it in the long term.

Indeed looks a little bit dead for now.

![image](https://github.com/bfenetworks/bfe/assets/28711504/0b98180a-6fde-4218-b7cf-b3c7372dfafb)

**Comment by mileszhang2016** [comment]
Thanks a lot for your kindness.
BFE project is still active.
The forwarding engine is changing slow，and BFE ingress is in developing (https://github.com/bfenetworks/ingress-bfe).
The new gateway api for BFE is preparing now, and will be released in 2023 Q3.
